amazon/amazon.json

```diff
@@ -0,0 +1,45 @@
+{
+	"variables": {
+		"iso_url": "http://mirror.chpc.utah.edu/pub/archlinux/iso/latest/archlinux-2017.10.01-x86_64.iso",
```

Comment:
Never messed with this repo so ignore if I'm talking crap, but, Arch?

Reply:
We are using the arch iso to install yum, so that we can just install amazon linux from the repositories, since it doesn't have an iso that we could kickstart from.

Arch has yum in the aur, which makes it an easy choice since the iso just boots to a shell.

Reply:
Arch FTW!!!!